sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory/src/main/resources/static/index.html

```diff
@@ -64,7 +64,7 @@ <h1>Todo List</h1>
 <script src="https://code.angularjs.org/1.2.25/angular-route.js"></script>
 <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
 <script src="App/Scripts/msal.js"></script>
```

Comment:
I remember that `msal.js` is not used in the application. Then can this line be deleted?

Reply:
we can delete `msal-angular.js` , but `msal.js` is used in my own function to obtaining tokens for "Authorization Code Flow with PKCE"

Reply:
I got it. Thank you for your explanation. 👍